examples/workspace_io/README.md

```
@@ -0,0 +1,163 @@
+# export OPENAI_BASE_URL="https://api.deepseek.com/v1"
+
+cd examples/workspace_io
+go run . \
+  -model deepseek-v4-flash \
+  -store ./skills_store \
+  -prompt "Say a short hello so I can verify the agent finished."
+```
+
+Expected output (abridged):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)